Sources/MapboxGeocoder/MapboxGeocoder.h

```diff
@@ -8,5 +8,5 @@ FOUNDATION_EXPORT double MapboxGeocoderVersionNumber;
 
 FOUNDATION_EXPORT const unsigned char MapboxGeocoderVersionString[];
 
-#import "MBPlacemarkPrecision.h"
-#import "MBPlacemarkScope.h"
+#import <MapboxGeocoder/MBPlacemarkPrecision.h>
+#import <MapboxGeocoder/MBPlacemarkScope.h>
```

Comment:
Fixes build related warning.

Reply:
I’m not sure if we need these import statements. If so, it would be for Objective-C, but it’s more likely to be the cause of issues like #59 in Objective-C. In any case, make sure to test in CocoaPods, which has a different header search path than under the normal Xcode build system.

Reply:
Looks like these import statements are needed for type-aliasing in Swift. Also checked whether we can build with Swift using `$ pod lib lint MapboxGeocoder.swift.podspec --verbose` and everything seems to work well.